### PR TITLE
`showIf` binding method

### PR DESCRIPTION
**Pull request by Starzu**

### CODE REVIEW

File: core/frontend/src/main/scala/io/udash/bindings/Bindings.scala [hunk 1]
```diff
@@ -42,6 +42,19 @@ trait Bindings {
     new SimplePropertyModifier[T](property, true)
 
   /**
+    * Shows provided DOM elements onyl if property value is `true`.
```
Comment (ddworak) [inline core/frontend/src/main/scala/io/udash/bindings/Bindings.scala]: only

File: core/frontend/src/main/scala/io/udash/bindings/Bindings.scala [hunk 2]
```diff
@@ -42,6 +42,19 @@ trait Bindings {
...
+    *
+    * @param property Property to check.
+    * @param elements  Elements to show if property value is `true`.
+    * @return Modifier for bounded property.
+    */
+  def showIf(property: ReadableProperty[Boolean])(elements: Seq[dom.Element]) =
+    new PropertyModifier[Boolean](property, {
+      case true => elements
+      case false => Seq(emptyStringNode())
```
Comment (ddworak) [inline core/frontend/src/main/scala/io/udash/bindings/Bindings.scala]: Wouldn't `Seq.empty` work just as well?

Comment (Starzu) [inline core/frontend/src/main/scala/io/udash/bindings/Bindings.scala]: Right :)

**Comment by Starzu** [comment]
@ddworak updated, take a look

**Comment by ddworak** [comment]
lgtm

**Pull request merged**